### PR TITLE
Dev db

### PR DESCRIPTION
**Pull request by BrettMatsumoto**

### CODE REVIEW

File: server/database/seeds/itemImages.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  return knex('table_name').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('table_name').insert([
+        {id: 1, colName: 'rowValue1'},
+        {id: 2, colName: 'rowValue2'},
+        {id: 3, colName: 'rowValue3'}
+      ]);
+    });
+};
```
Comment (lukefiorio) [inline server/database/seeds/itemImages.js]: Populate with actual data & tablename

File: server/database/seeds/items.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+          status_id: 1,
+          condition_id: 5,
+          category_id: 2,
+          subCategory_id: 1,
+          name: "Skechers Men's Go Walk Evolution Ultra-Impeccable Sneaker",
+          quantity: 4,
+          dimensions: 'Size 7',
+          viewCount: 17,
+          price: 17.17,
+          description: 'kick ass 55 inch tv bruh',
```
Comment (lukefiorio) [inline server/database/seeds/items.js]: update description to shoes

File: server/database/seeds/items.js [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+          approved: true,
+        },
+        {
+          user_id: 3,
+          status_id: 2,
+          condition_id: 6,
```
Comment (lukefiorio) [inline server/database/seeds/items.js]: there are only 5 conditions

File: server/database/seeds/items.js [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+          category_id: 2,
+          subCategory_id: 1,
+          name: 'Adidas Womens Questar Slide',
+          quantity: 1,
+          dimensions: 'Size 5',
+          viewCount: 909,
```
Comment (lukefiorio) [inline server/database/seeds/items.js]: would it have any views if it "approved": false ?

File: server/database/seeds/items.js [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+          status_id: 2,
+          condition_id: 6,
+          category_id: 3,
+          subCategory_id: 2,
+          name: 'Almost Paradise',
+          quantity: 1,
+          dimensions: '626 pages',
+          viewCount: 17,
+          price: 17.17,
+          description: 'Its about Guam',
```
Comment (lukefiorio) [inline server/database/seeds/items.js]: apostophre It's

File: server/database/seeds/items.js [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+          price: 179.99,
+          description: 'lightly damaged but still works',
+          approved: false,
+        },
+        {
+          user_id: 3,
+          status_id: 1,
+          condition_id: 5,
+          category_id: 2,
+          subCategory_id: 1,
```
Comment (lukefiorio) [inline server/database/seeds/items.js]: should be subCategory_id: 3

File: server/database/seeds/items.js [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+          price: 17.17,
+          description: 'kick ass 55 inch tv bruh',
+          approved: true,
+        },
+        {
+          user_id: 3,
+          status_id: 2,
+          condition_id: 6,
+          category_id: 2,
+          subCategory_id: 1,
```
Comment (lukefiorio) [inline server/database/seeds/items.js]: should be subCategory_id: 3

File: server/database/seeds/items.js [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+          price: 32.91,
+          description: 'flippy flops',
+          approved: false,
+        },
+        {
+          user_id: 4,
+          status_id: 1,
+          condition_id: 1,
+          category_id: 2,
+          subCategory_id: 2,
```
Comment (lukefiorio) [inline server/database/seeds/items.js]: should be subCategory_id: 3

File: server/database/seeds/items.js [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+          price: 12.22,
+          description: 'You will look good in this',
+          approved: true,
+        },
+        {
+          user_id: 4,
+          status_id: 2,
+          condition_id: 2,
+          category_id: 2,
+          subCategory_id: 2,
```
Comment (lukefiorio) [inline server/database/seeds/items.js]: should be subCategory_id: 4

File: server/database/seeds/items.js [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+          price: 4.86,
+          description: 'This shirt got me laid',
+          approved: false,
+        },
+        {
+          user_id: 5,
+          status_id: 1,
+          condition_id: 3,
+          category_id: 3,
+          subCategory_id: 1,
```
Comment (lukefiorio) [inline server/database/seeds/items.js]: should be subCategory_id: 5 or 6

File: server/database/seeds/items.js [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+          price: 12.0,
+          description: 'A lifechanging story',
+          approved: true,
+        },
+        {
+          user_id: 5,
+          status_id: 2,
+          condition_id: 4,
+          category_id: 3,
+          subCategory_id: 1,
```
Comment (lukefiorio) [inline server/database/seeds/items.js]: should be subCategory_id: 5 or 6

File: server/database/seeds/items.js [hunk 12]
```diff
@@ -0,0 +1,178 @@
...
+          price: 10.0,
+          description: 'Visit high-tech battlefields, surreal other worlds, an orbital habitat, and a distant moon.',
+          approved: false,
+        },
+        {
+          user_id: 6,
+          status_id: 1,
+          condition_id: 5,
+          category_id: 3,
+          subCategory_id: 2,
```
Comment (lukefiorio) [inline server/database/seeds/items.js]: should be subCategory_id: 5 or 6

File: server/database/seeds/items.js [hunk 13]
```diff
@@ -0,0 +1,178 @@
...
+          description: 'A book about thinning air',
+          approved: true,
+        },
+        {
+          user_id: 6,
+          status_id: 2,
+          condition_id: 6,
+          category_id: 3,
+          subCategory_id: 2,
+          name: 'Almost Paradise',
```
Comment (lukefiorio) [inline server/database/seeds/items.js]: should be subCategory_id: 5 or 6

File: server/database/seeds/itemImages.js (hunk 1 above)
Comment (BrettMatsumoto) [reply server/database/seeds/itemImages.js]: All images have a stock test image.

File: server/database/seeds/users.js [hunk 14]
```diff
@@ -0,0 +1,19 @@
...
+const bcrypt = require('bcryptjs');
+const saltRounds = 12;
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('users').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('users').insert([
+        {active: true, role_id: 1, theme_id: 1, username: '01_Admin', password: bcrypt.hash('password', saltRounds), name: 'Jack Sparrow', email: 'seaturtules@email.com', profileImageUrl: 'filler'},
```
Comment (lukefiorio) [inline server/database/seeds/users.js]: can we not have usernames start with non-letters?

Comment (lukefiorio) [reply server/database/seeds/users.js]: add actual image link to profileImageUrl

Comment (lukefiorio) [reply server/database/seeds/users.js]: same for all below

File: server/database/seeds/items.js (hunk 2 above)
Comment (BrettMatsumoto) [reply server/database/seeds/items.js]: Updated

File: server/database/seeds/paymentCards.js [hunk 15]
```diff
@@ -0,0 +1,16 @@
...
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('paymentCards').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('paymentCards').insert([
+        {cardNumber: '1111-1111-1111-1111', expiration: '12/20', cvv: '123', street: '321 Easy Street', aptSuite: '', city: 'New York City', state: 'NY', country: 'USA', zip: '12345', user_id: 1},
+        {cardNumber: '2222-2222-2222-2222', expiration: '11/20', cvv: '456', street: '1155 Buckey Road', aptSuite: '', city: 'Los Angeles', state: 'CA', country: 'USA', zip: '12345', user_id: 2},
+        {cardNumber: '3333-3333-3333-3333', expiration: '10/20', cvv: '789', street: '1428 Elm Street', aptSuite: '', city: 'Chicago', state: 'IL', country: 'USA', zip: '12345', user_id: 3},
+        {cardNumber: '4444-4444-4444-4444', expiration: '9/20', cvv: '012', street: '307A Kamani Street', aptSuite: '', city: 'Housten', state: 'TX', country: 'USA', zip: '12345', user_id: 4},
```
Comment (lukefiorio) [inline server/database/seeds/paymentCards.js]: Houston

File: server/database/seeds/items.js (hunk 3 above)
Comment (BrettMatsumoto) [reply server/database/seeds/items.js]: Adjusted condition 6 to be condition 1's

File: server/database/seeds/categories.js [hunk 16]
```diff
@@ -0,0 +1,13 @@
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('categories').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('categories').insert([
+        {categoryName: 'Electronics'},
+        {categoryName: 'Clothing'},
```
Comment (lukefiorio) [inline server/database/seeds/categories.js]: Apparel... since shoes are not really "clothing" per se

File: server/database/seeds/items.js (hunk 4 above)
Comment (BrettMatsumoto) [reply server/database/seeds/items.js]: I was working under the assumption that all items being posted were approved until flagged for being inappropriate/vulgar and then set as approved: false from there.

File: server/database/models/ShippingAddress.js [hunk 17]
```diff
@@ -0,0 +1,17 @@
+const bookshelf = require('../bookshelf');
+
+require('./User');
+class ShippingAddress extends bookshelf.Model {
+  get tableName() {
+    return 'ShippingAddress';
```
Comment (lukefiorio) [inline server/database/models/ShippingAddress.js]: return 'shippingAddresses

File: server/database/seeds/items.js (hunk 5 above)
Comment (BrettMatsumoto) [reply server/database/seeds/items.js]: Adjusted for apostrophe

File: server/database/seeds/items.js (hunk 6 above)
Comment (BrettMatsumoto) [reply server/database/seeds/items.js]: adjusted

**Pull request merged**